src/presence_analyzer/templates/base.html

```diff
@@ -0,0 +1,52 @@
+<!doctype html>
+<html lang=en>
+<head>
+    <meta charset=utf-8>
+    {% block title %}{% endblock %}
+    <meta name="keywords" content="" />
+    <meta name="description" content=""/>
+    <meta name="author" content="STX Next sp. z o.o."/>
+    <meta name="viewport" content="width=device-width; initial-scale=1.0">
+    <link href="{{ url_for('static', filename='css/normalize.css') }}" media="all" rel="stylesheet" type="text/css" />
+    <link href="{{ url_for('static', filename='css/main.css') }}" media="all" rel="stylesheet" type="text/css" />
+
+     <script src="{{ url_for('static', filename='js/jquery.min.js') }}"></script>
+    <script type="text/javascript" src="https://www.google.com/jsapi"></script>
+    {% block head %}{% endblock %}
+</head>
+
+<body>
+<div id="main">
+        <div id="header">
+            <h1>Presence analyzer</h1>
+            <ul>
+            {%- for name, link in pages.items() -%}
+                {%- if link == '/'+request.url.rsplit('/', 1)[1] -%}
```

Comment:
What for you need to add this leading slash?
Do it without additional  param in rsplit, it's not needed, and select the last element, no the second one.
And that if should only add the class, not generate the whole `<li>`.